docker/config/docker_common.env

```diff
@@ -0,0 +1,83 @@
+# Secrets and environment-specific variables for the developer docker
+# environment.
+
+# NOTE: These shouldn't be used on -dev, -stage, or -prod environments.
+
+# postgres
+database_hostname=postgresql
+database_port=5432
+database_username=postgres
+database_password=aPassword
+database_superusername=postgres
+database_superuserpassword=aPassword
+
+resource.postgresql.database_hostname=postgresql
+resource.postgresql.database_port=5432
+secrets.postgresql.database_username=postgres
+secrets.postgresql.database_password=aPassword
+
+# resource.postgresql.database_replication_hostname=
+# secrets.postgresql.database_replication_password=
+# secrets.postgresql.database_replication_username=
+
+# postgres things for alembic
+# FIXME - sqlalchemy.url=postgresql://postgres:aPassword@postgresql:5432/socorro_migration_test
+
+# statsd
+resource.statsd.statsd_host=statsd
+
+# sentry
+# secrets.sentry.dsn=
+
+# rabbitmq things
+resource.rabbitmq.host=rabbitmq
+resource.rabbitmq.virtual_host=rabbitvhost
+secrets.rabbitmq.rabbitmq_user=rabbituser
+secrets.rabbitmq.rabbitmq_password=rabbitpwd
+
+# elasticsearch things
+elasticsearch_url=http://elasticsearch:9200
+resource.elasticsearch.elasticsearch_urls=http://elasticsearch:9200
+resource.elasticsearch.elasticSearchHostname=elasticsearch
+
+# boto things
+resource.boto.host=localstack-s3
+resource.boto.port=5000
+resource.boto.secure=false
+resource.boto.access_key=foo
+secrets.boto.secret_access_key=foo
+resource.boto.bucket_name=dev_bucket
+
+# webapp things
+ALLOWED_HOSTS=localhost
+
+AWS_ACCESS_KEY=foo
+AWS_SECRET_ACCESS_KEY=foo
+CACHE_LOCATION=socorro-cache.mocotoolsprod.net:11211
+DATABASE_URL=postgres://postgres:aPassword@postgresql:5432/breakpad
+DATABASE_USER=postgres
+DATASERVICE_DATABASE_HOSTNAME=postgresql
+DATASERVICE_DATABASE_PASSWORD=aPassword
+DATASERVICE_DATABASE_PORT=5432
+DATASERVICE_DATABASE_USERNAME=postgres
+ELASTICSEARCH_URLS=http://elasticsearch:9200
+GOOGLE_ANALYTICS_ID=
+OAUTH2_CLIENT_ID=
+OAUTH2_CLIENT_SECRET=
+RABBITMQ_HOST=rabbitmq
+RABBITMQ_USER=rabbituser
+RABBITMQ_PASSWORD=rabbitpwd
+RABBITMQ_VIRTUAL_HOST=rabbitvhost
+RAVEN_DSN=
+SECRET_KEY=secretkey
+SESSION_COOKIE_SECURE=False
+SYMBOLS_BUCKET_DEFAULT_NAME=
+SYMBOLS_BUCKET_EXCEPTIONS=
+
+# Put the static files for the webapp in /tmp/crashstats-static/ for
+# development because then it doesn't create permissiosn issues between the
+# container and the host.
+STATIC_ROOT=/tmp/crashstats-static/
+
+PORT=8000
+NUM_WORKERS=1
```

Comment:
Don't need these two things since these are for local development and we're not using uwsgi anymore for that.